apiserver/facades/client/modelmanager/shims.go

```diff
@@ -50,6 +58,16 @@ func (s spaceStateShim) ConstraintsBySpaceName(name string) ([]space.Constraints
 	return results, nil
 }
 
+// TODO(nvinuesa): This method is needed temporarily and must be removed when
+// the new spaces domain has been finished and the mongodb state removed.
```

Comment:
Do we need the TODO here? Once we move to the domain, I'm hoping this spaceStateShim could go away.

Reply:
you are right, done